catroid/config/pmd.xml

```diff
@@ -77,4 +77,15 @@
     </rule >
 
     <rule ref="rulesets/java/unusedcode.xml" />
+
+    <rule ref="rulesets/java/junit.xml">
+        <exclude name="JUnitTestContainsTooManyAsserts" />
+        <exclude name="JUnitAssertionsShouldIncludeMessage" />
+        <exclude name="JUnitTestsShouldIncludeAssert" />
+
+        <exclude name="TestClassWithoutTestCases">
+            <exclude-pattern>.*src\androidTest\java\org\catrobat\catroid\test\physics\PhysicsBaseTest.java</exclude-pattern>
+            <exclude-pattern>.*src\androidTest\java\org\catrobat\catroid\uitest\ui\activity\DoubleClickOpensViewOnceTest.java</exclude-pattern>
+        </exclude>
```

Comment:
Instead of excluding those files here, please add pmds ignore syntax to the files itself.
e.g. `@SuppressWarnings("PMD.UnusedLocalVariable")`

_DoubleClickOpensViewOnceTest_ is a Robotium test and will be removed soon.
Inheriting from tests, as done in _PhysicsBaseTest_, is bad practice anyway.